ui/src/app/cohort-search/list-search/list-search.component.tsx

```diff
@@ -284,9 +285,8 @@ export const ListSearch = fp.flow(withCdrVersions(), withCurrentWorkspace(), wit
       };
     }
     componentDidMount(): void {
-      const {cdrVersionListResponse, searchTerms, searchContext: {source}, workspace: {cdrVersionId}} = this.props;
-      const cdrVersions = cdrVersionListResponse.items;
-      this.setState({cdrVersion: cdrVersions.find(cdr => cdr.cdrVersionId === cdrVersionId)});
```

Comment:
this `find` is a little more complicated with the new response structure so I moved it into a utils function instead of copying the new logic everywhere